autobahn/wamp_session.ipp

```diff
@@ -772,6 +772,29 @@ void wamp_session<IStream, OStream>::process_welcome(const wamp_message& message
 }
 
 template<typename IStream, typename OStream>
+void wamp_session<IStream, OStream>::process_abort(const wamp_message& message)
+{
+    // [ABORT, Details|dict, Reason|uri]
+    
+    if (message.size() != 3 ) {
+        throw protocol_error("ABORT - length must be 3");
```

Comment:
Yes they should be propagated to the client API via the corresponding m_session_join future.